gerousiabot/discord_apis.py

```diff
@@ -2,7 +2,57 @@
 This module will hold the functions used for
 interacting with the Discord APIs
 """
+import hikari
+import asyncio
+import os
+from dotenv import load_dotenv
 
+def get_api_token(key) -> str:
+    """
+    Gets the API token for a given key
+    :param key: the key for the token you want returned
+    :return: the API token
+    """
+    load_dotenv()
+    return os.getenv(key)
+
+def get_rest_client():
+    rest_app = hikari.RESTApp()
+    token = get_api_token("DISCORD_API_KEY")
+    rest_client = rest_app.acquire(token, "Bot")
```

Comment:
[Question] Is the argument `"bot"` arbitrary?

Reply:
no. so the REST Client can work for a User or a Bot if the "Bot" argument isn't there then it set up the Auth token as a BEARER 

See Authorization section
https://discord.com/developers/docs/reference